src/powersystem.jl

```diff
@@ -254,8 +256,8 @@ function bustypeindex(bus, gen)
     gencon = zeros(Int8, size(bus, 1))
 
     for i in 1:size(gen, 1)
-        if gen[i, GEN_STAT] == 1
-            gencon[gen[i, GEN_BUS]] += 1
+        if gen[i, GEN_STATUS] == 1
+            gencon[Int(gen[i, GEN_BUS])] += 1
```

Comment:
Why is the array `gen` not of type `AbstracArray{Int}`?

Reply:
I am using {Any} array now instead of Floats64. Matpower uses Float64 for the whole array.

I think we should use Dicts instead.